core/src/test/scala/unit/kafka/cluster/PartitionTest.scala

```diff
@@ -954,7 +955,7 @@ class PartitionTest extends AbstractPartitionTest {
       // Append records to partitions, one partition-per-thread
       val futures = partitions.map { partition =>
         executor.submit(CoreUtils.runnable {
-          (1 to 10000).foreach { _ => partition.appendRecordsToLeader(createRecords(baseOffset = 0), isFromClient = true) }
+          (1 to 10000).foreach { _ => partition.appendRecordsToLeader(createRecords(baseOffset = 0), origin = AppendOrigin.Client, requiredAcks = 0) }
```

Comment:
Merge conflict here due to upstream change that switches CoreUtils.runnable for a scala lambda used as a runnable.